adsingestp/parsers/jats.py

```diff
@@ -899,9 +899,10 @@ def citation_context(
         :param text: text of fulltext XML to parse
         :param bsparser: parser to use with BeautifulSoup
         :param input_bibcode: string, bibcode of input XML, if known # TODO do I need this? do I need to resolve and return the paper's own bibcode?
-        :param num_char: integer, check that citation paragraph is at least this long; if it's shorter, return the
+        :param num_char: integer, check that citation paragraph is at least this long; if it's shorter, return the. Note if output is BeautifulSoup oblects set num_char to 1
```

Comment:
Why does `num_char` need to be set to 1? I don't remember the logic for this part

Reply:
It was so that if beautifulsoup objects are the desired output, then the section where the previous paragraph and next paragraph are extracted is skipped.  I tweaked the order of the `if` statement on line 931 so this keyword is no longer necessary for beautifulsoup output.